com.unity.perception/Runtime/GroundTruth/Labelers/AnimationPoseLabel.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    /// <summary>
+    /// Record that maps a pose to a timestamp
+    /// </summary>
+    [Serializable]
+    public class PoseTimestampRecord
+    {
+        /// <summary>
+        /// The duration to use for this timestamp in seconds
+        /// </summary>
+        public float duration;
```

Comment:
My suggestion would be to change this duration value to something like startTime or startingKeyFrame. Users would first author a Pose timestamp that starts at 0 then add timestamps with increasing time values at each animation transition point.